cmake/pcl_utils.cmake

```diff
@@ -60,7 +60,7 @@ endmacro(REMOVE_VTK_DEFINITIONS)
 # Pull the component parts out of the version number.
 macro(DISSECT_VERSION)
     # Detect if we're in a developlment version and generate pretty version string
-    if(${PCL_VERSION_TWEAK} EQUAL 99)
+    if(${PCL_VERSION_TWEAK} AND ${PCL_VERSION_TWEAK} EQUAL 99)
```

Comment:
I think the right way to fix the problem Tsukasa experienced is to simply drop variable evaluation, i.e.:

```cmake
     if(PCL_VERSION_TWEAK EQUAL 99) 
```

Why? Variable evaluation with `${}` applies before the `if` command even receives the arguments. Therefore, if tweak version is empty, the current code appears to the `if` command as `if( EQUAL 99)`, which is not a valid code.

Reply:
Thanks for the tip. I was not aware of that detail concerning evaluation order.